tasks/revoke-sudo.yml

```diff
@@ -0,0 +1,10 @@
+- name: Get groups for user '{{ user }}'
+  command: "groups {{ user }}"
+  register: current_groups
+  changed_when: false
+
+- name: Revoke 'sudo' for '{{ user }}'
+  user:
+    name: "{{ user }}"
+    groups: "{{ current_groups.stdout | replace(user + ' : ', '') | replace('sudo', '') | split }}"
```

Comment:
I don't get this. What's the output of `current_groups.stdout`? Why is `' : '` added to user?

Reply:
this is how the output looks:

```
$ sudo groups andy
andy : andy sudo docker accounts
```

It's quite different compared to executing `groups` without any username.

```
$ groups
andy sudo docker accounts
```

Reply:
It doesn't do that on my system:
```
> sudo groups flo
wheel audio input lp storage users network power docker flo
```
So I guess this is something debian specific?
Maybe there is an option to make it output in a more friendly format?

Reply:
Maybe use `id -nG flo` instead?

Reply:
addressed with [81b69e0](https://github.com/simplificator/ansible-role-linux-accounts/pull/11/commits/81b69e0616e0f6672a3c64cf411aca37390a2a86).

Reply:
Ok, this looks a lot cleaner now